src/style/mixins.scss

```
@@ -602,3 +602,19 @@ $narrow-dialog-breakpoint: 760px;
         @content;
     }
 }
+
+@mixin table-row-indicator($border-color, $z-index) {
```

```suggestion
@mixin table-row-selected-indicator($border-color, $z-index) {
```

Since the mixin is only responsible for visualizing the row that is _selected_, it's better to name it this way. Otherwise, the reader can be confused about what this piece of code actually does.